### PR TITLE
chore(docs): adds the whats new section (VIV-1594)

### PR DESCRIPTION
**Pull request by TaylorJ76**

### CODE REVIEW

File: docs/whats-new/monthly-updates/march-2024.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## New Vivid "Office Hours" Meeting
+
+We are introducing a regular meeting (every 2 weeks) where you, the consumers of the design system, can connect with the Vivid team to ask questions, raise or discuss feature requests, share concerns or just chat and connect.
+
+We have an [agenda document](https://docs.google.com/document/d/1E0yvyGUzBoQFH5l_W6ElBoZaxqZ3HWmDLDqOl0lc8a0/edit?usp=sharing) where we encourage you to add topics / questions that you'd like to raise in the meeting. By doing this, everybody gets a view of what will be discussed and it also gives the Vivid team a little time to prepare prior to the meeting.
+
+Topics will be discuss at high level and timeboxed to 5 - 10 mins. The aim of this meeting is not to always resolve every query, but to kick off further, more detailed discussions with interested parties.
+
+Attendence is not mandatory. You don't have to ask question to attend, it's fine to just listen in and observe. Also, feel free to dip in and out of the meeting, we don't want to take up anymore of your time than neccessary.
```
Comment (RichardHelm) [inline docs/whats-new/monthly-updates/march-2024.md:11]:
```suggestion
Attendance is not mandatory. You don't have to ask question to attend, it's fine to just listen in and observe. Also, feel free to dip in and out of the meeting, we don't want to take up any more of your time than necessary.
```

File: docs/whats-new/monthly-updates/march-2024.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# March 2024
+
+## New Vivid "Office Hours" Meeting
+
+We are introducing a regular meeting (every 2 weeks) where you, the consumers of the design system, can connect with the Vivid team to ask questions, raise or discuss feature requests, share concerns or just chat and connect.
+
+We have an [agenda document](https://docs.google.com/document/d/1E0yvyGUzBoQFH5l_W6ElBoZaxqZ3HWmDLDqOl0lc8a0/edit?usp=sharing) where we encourage you to add topics / questions that you'd like to raise in the meeting. By doing this, everybody gets a view of what will be discussed and it also gives the Vivid team a little time to prepare prior to the meeting.
+
+Topics will be discuss at high level and timeboxed to 5 - 10 mins. The aim of this meeting is not to always resolve every query, but to kick off further, more detailed discussions with interested parties.
```
Comment (RichardHelm) [inline docs/whats-new/monthly-updates/march-2024.md:9]:
```suggestion
Topics will be discussed at a high level and timeboxed to 5 - 10 mins. The aim of this meeting is not to always resolve every query, but to kick off further, more detailed discussions with interested parties.
```

File: docs/whats-new/roadmap.md [hunk 3]
```diff
@@ -81,7 +81,7 @@
   </vwc-data-grid-row>
   <vwc-data-grid-row>
     <vwc-data-grid-cell>
-      Video Player
+      <a href="/components/video-player">Video Player</a>
```
Comment (RichardHelm) [inline docs/whats-new/roadmap.md:84]:
```suggestion
      <a href="/components/video-player/">Video Player</a>
```

File: docs/whats-new/roadmap.md [hunk 4]
```diff
@@ -90,7 +90,7 @@
       <vwc-button label="Done" icon="check" target="_blank" href="https://www.figma.com/file/tWEyQlBY6cBymajaJPLSy0/Vivid-3.0-WIP-Drafts?type=design&node-id=23%3A13702&mode=design&t=PGI1aODhsHS1YDhV-1" connotation="success"></vwc-button>
     </vwc-data-grid-cell>
     <vwc-data-grid-cell>
-      <vwc-button label="Exp: 03/2024" icon="more-horizontal-line" target="_blank" href="https://jira.vonage.com/browse/VIV-1486" connotation="cta"></vwc-button>
+      <vwc-button  label="Awaiting AT" icon="more-horizontal-line" href="/components/video-player" connotation="success"></vwc-button>
```
Comment (RichardHelm) [inline docs/whats-new/roadmap.md:93]:
```suggestion
      <vwc-button label="Awaiting AT" icon="more-horizontal-line" href="/components/video-player/" connotation="success"></vwc-button>
```

File: apps/docs/_data/whatisnew.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+[
+	{
+		"title": "Monthly Updates",
+		"markdown": "./docs/whats-new/monthly-updates/march-2024.md",
+		"children": ["March 2024", "February 2024"]
```
Comment (RichardHelm) [inline apps/docs/_data/whatisnew.json:5]:
```suggestion
		"children": ["March 2024"]
```

File: apps/docs/_includes/navigation.njk [hunk 6]
```diff
@@ -4,6 +4,23 @@
     {% endset %}
     <script>{{ componentsFilter | safe }}</script>
     <vwc-nav class="side-nav">
+        <vwc-nav-disclosure label="What's New" {% if page.url.includes('/whats-new/') %} open aria-current="true" {% endif %}>
```
Comment (RichardHelm) [inline apps/docs/_includes/navigation.njk:7]: I think this section should be further down

**Review by RichardHelm — CHANGES_REQUESTED** [review]

File: apps/docs/_includes/navigation.njk (hunk 6 above)
Comment (TaylorJ76) [reply apps/docs/_includes/navigation.njk:7]: I don't feel too strongly on this, but most visitors to the site are returning and are already set up. 
News and updates are more relevant to them than getting started.

**Review by RichardHelm — APPROVED** [review]

File: apps/docs/_includes/navigation.njk (hunk 6 above)
Comment (RichardHelm) [reply apps/docs/_includes/navigation.njk:7]: Not sure about this, but approved for now 👍

**Pull request merged**